spec/features/workflow/unwithdraw_without_managing_editor_permissions.spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+RSpec.feature "Undo a withdraw without managing editor permission" do
+  background do
+    given_i_dont_have_the_managing_editor_permission
+  end
```

Comment:
Rather than having this in a background block I think this would make sense in the scenario since it's not used to apply to multiple scenarios